### PR TITLE
Added EnergyIQ API calls based on node-tado-client v0.13.3

### PR DESCRIPTION
**Pull request by ViTra86**
Hi Matt, 

I tried adding the Energy IQ calls to the node-red project but never did anything with node-red before so not sure if this is the way it's suppose to be. Please let me know if I need to change anything.

### CODE REVIEW

File: tado.js [hunk 1]
```diff
@@ -2,193 +2,229 @@
...
+				case "setOpenWindowMode":
+					call(arg("homeId"), arg("zoneId"), bool(arg("openWindowMode")));
+					break;
+					
+				case "addEnergyIQTariff":
+					call(arg("homeId"), arg("unit"), arg("startDate"), arg("endDate"), Number(arg("tariffInCents")));
+					break;
+
+				case "updateEnergyIQTariff":
+					//console.log(node);
```
Comment (mattdavis90) [inline tado.js:209]: Can you remove the log message

File: tado.js [hunk 2]
```diff
@@ -2,193 +2,229 @@
...
+					call(arg("homeId"), arg("tariffId"), arg("unit"), arg("startDate"), arg("endDate"), Number(arg("tariffInCents")));
+					break;
+					
+				case "addEnergyIQMeterReading":
+					call(arg("homeId"), arg("readingDate"), parseInt(arg("reading"), 10));
+					break;
+					
+				case "deleteEnergyIQMeterReading":
+				console.log(node);
```
Comment (mattdavis90) [inline tado.js:218]: and this one please

File: tado.js [hunk 3]
```diff
@@ -2,193 +2,229 @@
...
+					call(arg("homeId"), arg("zoneId"), bool(arg("openWindowMode")));
+					break;
+					
+				case "addEnergyIQTariff":
+					call(arg("homeId"), arg("unit"), arg("startDate"), arg("endDate"), Number(arg("tariffInCents")));
+					break;
+
+				case "updateEnergyIQTariff":
+					//console.log(node);
+					call(arg("homeId"), arg("tariffId"), arg("unit"), arg("startDate"), arg("endDate"), Number(arg("tariffInCents")));
```
Comment (mattdavis90) [inline tado.js:210]: Could this be a `parseFloat` or similar rather than constructing a `Number`

File: tado.js [hunk 4]
```diff
@@ -2,193 +2,229 @@
...
+						node.send(msg);
+					}
+
+					if (done) {
+						done();
+					}
+				}).catch(err => {
+					node.status({ fill: "red", shape: "ring", text: "errored" });
+					if (done) {
+						//console.log(err);
```
Comment (mattdavis90) [inline tado.js:104]: This log can be removed as well, please

File: tado.html [hunk 5]
```diff
@@ -400,8 +471,8 @@ <h3>References</h3>
                         home.show();
                         device.show();
                         geoTracking.show();
-                        break;
-
+                        break;					
+
```
Comment (mattdavis90) [inline tado.html:475]: Trailing whitespace on these lines?

**Review by mattdavis90 — CHANGES_REQUESTED** [review]
I've added some online comments. I think this looks about right though. I'm on my mobile so can't test right now. Did you load it up in NodeRed?

**Comment by ViTra86** [comment]
Made changes according to your notes. Yes I did load it up in NodeRed and tested all calls succesfull.

**Comment by mattdavis90** [comment]
Great, thanks. I'll load it up when I'm back home and just double check everything then think this is good to merge. Thanks for the PR

**Comment by mattdavis90** [comment]
Released in v0.10.5

**Pull request merged**